test/Main.purs

```diff
@@ -28,182 +26,232 @@ eqAbsolute' = eqAbsolute (Tolerance 0.1)
 
 infix 1 eqAbsolute' as =~=
 
-main ∷ Effect Unit
-main = runTest do
+main :: Effect Unit
+main = do
 
 
-  suite "Data.Number.fromString" do
-    test "valid number string" do
-      assert "integer strings are coerced" $
-        fromMaybe false $ map (_ == 123.0) $ fromString "123"
+  -- suite "Data.Number.fromString" do
```

Comment:
Rather than commenting these out (I think checking in commented-out code is usually best avoided), how about replacing these with calls to `log`? That way you’ll still have the context in the test output.

Reply:
Sounds good. I'll work on that.

Reply:
Done.